packages/repository/src/repositories/kv.repository.bridge.ts

```diff
@@ -11,6 +11,7 @@ import {Entity} from '../model';
 import {KeyValueRepository, KeyValueFilter} from './kv.repository';
 
 import {juggler, ensurePromise} from './legacy-juggler-bridge';
+import {DeepPartial} from 'loopback-datasource-juggler';
```

Comment:
This isn't used anywhere in the file. Is it needed?

Reply:
It was by mistake.